hadoop-plugin/src/main/groovy/com/linkedin/gradle/azkaban/AzkabanHelper.groovy

```diff
@@ -267,7 +270,23 @@ class AzkabanHelper {
           azkProject.azkabanUserName = input.toString();
         }
 
-        input = console.readLine("New Azkaban Zip task (run 'ligradle tasks' to find existing Zip tasks) [enter to accept '${azkProject.azkabanZipTask}']: ");
+        HadoopZipExtension hadoopZipExtension = project.extensions.getByName("hadoopZip");
+        Map<String, CopySpec> zipMap = hadoopZipExtension.getZipMap();
+
+        if (zipMap.isEmpty()) {
+          logger.lifecycle("No zips configured in the hadoopZip block");
+        } else {
+          logger.lifecycle("\nFound the following zips in the hadoopZip block:");
+          def counter = 1;
+          zipMap.each { String zipName, CopySpec copySpec ->
+            logger.lifecycle("${counter++}. ${zipName.replace("hadoopZip","").replace("azkaban","")} - Enter \"${zipName}HadoopZip\" to use this zip");
```

Comment:
I actually don't think you need to replace either hadoopZip or azkaban out of zipName - I think we should just show the exact name the user used in the hadoopZip block, e.g.

hadoopZip {
  zip("myAzkabanZip") {
    ...
  }
}

Then when they run the task, they should see something like:

1. myAzkabanZip - Enter "myAzkabanZipHadoopZip" to use this zip